### PR TITLE
Improve extensions build. 

### PR DESCRIPTION
**Pull request by bergquist**

### CODE REVIEW

File: pkg/services/sqlstore/team.go [hunk 1]
```diff
@@ -268,7 +268,16 @@ func GetTeamMembers(query *m.GetTeamMembersQuery) error {
 	query.Result = make([]*m.TeamMemberDTO, 0)
 	sess := x.Table("team_member")
 	sess.Join("INNER", "user", fmt.Sprintf("team_member.user_id=%s.id", x.Dialect().Quote("user")))
-	sess.Where("team_member.org_id=? and team_member.team_id=?", query.OrgId, query.TeamId)
+	//sess.Where("team_member.org_id=? and team_member.team_id=?", query.OrgId, query.TeamId)
```
Comment (torkelo) [inline pkg/services/sqlstore/team.go]: commented out line?

File: public/app/features/org/team_details_ctrl.ts [hunk 2]
```diff
@@ -1,15 +1,21 @@
...
+  newGroupId: string;
+  enterprise: boolean;
 
   /** @ngInject **/
   constructor(private $scope, private backendSrv, private $routeParams, navModelSrv) {
     this.navModel = navModelSrv.getNav('cfg', 'teams', 0);
     this.userPicked = this.userPicked.bind(this);
     this.get = this.get.bind(this);
+    this.newGroupId = '';
+    this.enterprise = config.enterprise;
```
Comment (torkelo) [inline public/app/features/org/team_details_ctrl.ts]: long term I think feature toggles might be better, will make it easier to introduce more licence levels than oss & enterprise where depending on the license level different features are enabled/disabled.  But not needed right now, just a thought.

**Review by torkelo — COMMENTED** [review]
just some comments :)

File: public/app/features/org/team_details_ctrl.ts (hunk 2 above)
Comment (bergquist) [reply public/app/features/org/team_details_ctrl.ts]: Agree.

**Pull request merged**